pkgs/development/python-modules/okonomiyaki/default.nix

```
@@ -0,0 +1,43 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, attrs
+, distro
+, jsonschema
+, six
+, zipfile2
+, hypothesis
+, mock
+, packaging
+, testfixtures
+}:
+
+buildPythonPackage rec {
+  pname = "okonomiyaki";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "enthought";
+    repo = pname;
+    rev = version;
+    hash = "sha256-eWCOuGtdjBGThAyu15aerclkSWC593VGDPHJ98l30iY=";
+  };
+
+  propagatedBuildInputs = [ distro attrs jsonschema six zipfile2 ];
+
+  postPatch = ''
+    substituteInPlace /build/source/okonomiyaki/runtimes/tests/test_runtime.py \
```

```
substitute(): ERROR: file '/build/source/okonomiyaki/runtimes/tests/test_runtime.py' does not exist
```

---

This is still an erorr

---

both `nix build .#python310Packages.okonomiyaki .#python311Packages.okonomiyaki` and `nixpkgs-review rev 215001` cant reproduce this, what command are you running to build?

---

This if from the darwin ofborg check below and I just noticed why this is broken.

> Did nix cache a broken source tree perhaps?

It can't there is no cache for the CI checks.

we can't use an absolute path here

```suggestion
    substituteInPlace okonomiyaki/runtimes/tests/test_runtime.py \
```